### PR TITLE
Instanceof renamed import types

### PR DESCRIPTION
**Pull request by fitzgen**

### CODE REVIEW

File: crates/shared/src/lib.rs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #[macro_use]
```
Comment (alexcrichton) [inline crates/shared/src/lib.rs]: It looks like this file accidentally became executable?

I'm actually really curious, I've seen this happen from time to time, do you know what might cause it?

File: crates/shared/src/lib.rs [hunk 2]
```diff
@@ -82,7 +82,8 @@ pub struct ImportStatic {
 
 #[derive(Deserialize, Serialize)]
 pub struct ImportType {
-    pub name: String,
+    pub rust_name: String,
```
Comment (alexcrichton) [inline crates/shared/src/lib.rs]: We actually don't need to change the name here I think, the generated JS bindings don't need to know about the rust name, right? (we just need to be sure to fill in the JS name when this struct is generated)

**Comment by alexcrichton** [comment]
r=me, but looks like a web-sys test is failing?

File: crates/shared/src/lib.rs (hunk 1 above)
Comment (fitzgen) [reply crates/shared/src/lib.rs]: Yeah, I've seen it too, and I have no idea why. It is possible a mistyped emacs key chord does it? I really don't know... I'm certainly not running `chmod +x`! :-p

File: crates/shared/src/lib.rs (hunk 2 above)
Comment (fitzgen) [reply crates/shared/src/lib.rs]: Ah good idea, and that way we don't need to wonder if we have tracked down all the potential bits where we use the wrong version!

**Comment by fitzgen** [comment]
> r=me, but looks like a web-sys test is failing?

Gah :( I couldn't run those tests because webdriver is hanging for me.

Also, it seems that the `WASM_BINDGEN_FIREFOX_BIN_PATH` var is not used anymore. I'll file some follow up issues.

**Pull request merged**